### PR TITLE
feat: Unified access entrance, only the dashboard port is exposed to …

### PR DESCRIPTION
**Pull request by gxthrj**
…the outside

Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Unified access entrance, only the dashboard port is exposed to the outside.

### CODE REVIEW

File: compose/dashboard_conf/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    include /etc/nginx/mime.types;
+
+    location / {
+        try_files $uri $uri/ /index.html;
+    }
+
+    location /apisix/admin {
+        proxy_pass http://manager:8080/apisix/admin;
+    }
+}
```
Comment (membphis) [inline compose/dashboard_conf/nginx.conf:21]: need a blank line at the end of file

File: compose/dashboard_conf/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,21 @@
+server {
+    listen 80;
+    # gzip config
+    gzip on;
```
Comment (membphis) [inline compose/dashboard_conf/nginx.conf:4]: why do we need to enable `gzip` by default? I think `off` is better for us.

only CDN need to open `gzip` by default.

Comment (juzhiyuan) [reply compose/dashboard_conf/nginx.conf:4]: It's ok to use gzip here

https://github.com/ant-design/ant-design-pro/blob/master/docker/nginx.conf#L4

File: compose/dashboard_conf/nginx.conf [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    root /usr/share/nginx/html;
+    include /etc/nginx/mime.types;
+
+    location / {
+        try_files $uri $uri/ /index.html;
+    }
+
+    location /apisix/admin {
+        proxy_pass http://manager:8080/apisix/admin;
```
Comment (juzhiyuan) [inline compose/dashboard_conf/nginx.conf:19]: if we need X-Forwarded-Proto here?

File: compose/dashboard_conf/nginx.conf (hunk 1 above)
Comment (gxthrj) [reply compose/dashboard_conf/nginx.conf]: done

File: compose/dashboard_conf/nginx.conf (hunk 2 above)
Comment (gxthrj) [reply compose/dashboard_conf/nginx.conf:4]: use ant-design default configuration temporarily

File: compose/dashboard_conf/nginx.conf (hunk 3 above)
Comment (gxthrj) [reply compose/dashboard_conf/nginx.conf:19]: I think `http` is enough  for intranet

**Pull request merged**